src/Stratis.FederatedPeg.Features.FederationGateway/Interfaces/ICrossChainTransferStore.cs

```diff
@@ -30,29 +29,32 @@ public interface ICrossChainTransferStore : IDisposable
         /// <remarks>
         /// The transfers are set to <see cref="CrossChainTransfer.Status"/> of <see cref="CrossChainTransferStatus.Partial"/>
         /// or <see cref="CrossChainTransferStatus.Rejected"/> depending on whether enough funds are available in the federation wallet.
+        /// New partial transactions are recorded in the wallet to ensure that future transactions will not
+        /// attempt to re-use UTXO's.
         /// </remarks>
         Task RecordLatestMatureDepositsAsync(IDeposit[] deposits);
 
         /// <summary>
         /// Returns all partial transactions still in need of signatures.
         /// </summary>
         /// <returns>An array of fully signed transactions.</returns>
-        Task<Transaction[]> GetPartialTransactionsAsync();
+        /// <remarks>
+        /// The caller can order this list by looking at the corresponsing transaction order in the wallet.
```

Comment:
Can't this method order the list or pass in an `order` default parameter - then the caller doesn't need to do the extra operation in this remake - unless I'm missing something.

Reply:
Thanks